index.html

```diff
@@ -2453,7 +2482,8 @@ <h4>
                 connection</var> is <a data-link-for=
                 "PresentationConnectionState">connected</a> or
                   <a data-link-for="PresentationConnectionState">connecting</a>,
-                  then <a>queue a task</a> to run the following steps:
+                  then <a>queue a Presentation API task</a> to run the
```

Comment:
Re-reading this, I'm wondering whether, here and in the "Handling a termination confirmation in a controlling user agent" algorithm, it could be worth expanding that "current realm" means the [relevant realm](https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-global) of *known connection* (that realm may not be the same for each of them in particular, and may not be the realm under which the algorithm runs).

One possibility could be to rewrite as:
```
<a>queue a global task</a> on the <a>presentation task source</a>
given <var>known connection</var>'s <a>relevant global object</a> to...
```

Reply:
That's correct, _known connection_ likely belongs to a different document and thus a different global object.